public/scripts/world-info.js

```diff
@@ -2920,22 +2907,58 @@ async function getWorldEntry(name, data, entry) {
                 setWIOriginalDataValue(data, uid, 'extensions.vectorized', false);
                 template.addClass('disabledWIEntry');
                 break;
+                */
+        }
+        await saveWorldInfo(name, data);
+
+    });
+
+    const entryKillSwitch = template.find('div[name="entryKillSwitch"]');
+    entryKillSwitch.data('uid', entry.uid);
+    entryKillSwitch.on('click', async function (event) {
+        const uid = entry.uid;
+        data.entries[uid].disable = !data.entries[uid].disable;
+        let isActive = !data.entries[uid].disable;
+        setWIOriginalDataValue(data, uid, 'enabled', isActive);
+        if (isActive) {
```

Comment:
This one and the one below can be shorter. We still using jQuery, so
```js
        template.toggleClass('disabledWIEntry', !isActive);
        entryKillSwitch.toggleClass('fa-toggle-off', !isActive);
        entryKillSwitch.toggleClass('fa-toggle-on', isActive);
```
Just a suggestion.